domestic/templates/domestic/topic_landing_pages/generic.html

```diff
@@ -14,10 +14,9 @@
 </div>
 {% endblock %}
 
-{% if page.slug|lower == "advice" %}
-{% url "core:signup" as link_url %}
+{% if page.slug|lower == "advice" and not request.user.is_authenticated %}
   <div class="container">
-    {% include 'components/learning_banner.html' with tag_text="New" heading="Turn advice into action" sub_heading="Get exporting lessons from trade experts" cta_text="Upskill now" cta_url=link_url cta_alt_text="Upskill now" %}
+    {% include 'components/learning_banner.html' with tag_text="New" heading="Turn advice into action" sub_heading="Get exporting lessons from trade experts" cta_text="Upskill now" cta_url="/learn/categories" cta_alt_text="Upskill now" %}
```

Comment:
Nit: I get we're going with hard-coded for now, but ideally we want the trailing slash after `categories`

```suggestion
    {% include 'components/learning_banner.html' with tag_text="New" heading="Turn advice into action" sub_heading="Get exporting lessons from trade experts" cta_text="Upskill now" cta_url="/learn/categories/" cta_alt_text="Upskill now" %}
```